astro_2.0/components/EditableContent/EditableContent.module.scss

```diff
@@ -30,11 +30,33 @@ $border-radius: 3px;
   line-height: 24px;
   min-height: 160px;
   padding: $gap;
-  li,
+
+  ul,
+  pre {
+    margin: 0;
+  }
+
   p {
+    margin: 0 0 10px;
+  }
+
+  p,
+  ul {
     font-size: 14px;
     line-height: 24px;
   }
+
+  pre {
+    background-color: var(--color-neutral-20);
+    font-family: monospace, monospace;
+    font-size: 12px;
+    margin: 4px 0;
+    padding: 5px 10px;
+  }
+
+  img {
+    max-width: 100%;
+  }
```

Comment:
It looks like this block is being repeated multiple times. Can we put it in mixin and reuse it, e.g. like first example [here](https://sass-lang.com/documentation/at-rules/mixin)?

Reply:
fixed